### DISCUSSION TITLE
Copilot vim plugin: "Thank you for using GitHub Copilot..." message never disappear

### DISCUSSION THREAD

**Discussion by tony-go** [community/community#30493] [Copilot Conversations] [answered]
Hi 👋 

I subscribed to the copilot, I use it through the vim plugin.

Each time I switch from one file to another, I get this message:

```text
Thank you for using GitHub Copilot. Your extended access ends in one day.

Request Actions:
1. Copilot Settings
2. Dismiss
Type number and <Enter> or click with the mouse (q or empty cancels):
```

When I chose "1", the "billing" page appears, and nothing. (I see my subscription to copilot on the page).

I also try to sign out and set up again, but the message still pop-up every three actions I do on vim.

**Comment by tilmann1** [comment, marked as answer]
Can you please update to the latest version of the Neovim plugin and try again? Thanks!
